schedule/yam/agama_interactive.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
why not just agama? is taken? can we replace it?
s/nteractive/interactive/
In theory interactive is the default, so no need to mention.

Reply:
Updated it to agama.yaml.